### DISCUSSION TITLE
email forwared issue

### DISCUSSION THREAD

**Discussion by netpita** [postalserver/postal#2930] [Help with using Postal]
Hello.

I'm using postalserver for a couple of weeks and everything works just fine, except email forward.

I have an email server with email accounts from several domains, that uses my postalserver as a smarthost for outgoing email. And it works.
The problem is when an email account has a forward created (to forward all email to an external email address, for instance, gmail). The account's email server tries to resend the email through the postalserver but the postalserver responds with a 530 From/Sender name is not valid (in reply to end of DATA command))

Any idea or trick?
Am I missing something?

Thanks!

**Comment by willpower232** [comment]
I presume you need to turn on the sender header setting as the content of the message is what is invalidating the authentication but why do you need Postal as a smarthost? What benefit is it adding to this situation?

**Comment by netpita** [comment]
Hi, Will.
Thank you for your answer.

I already have Sender Header turned on (which enables Send as Any). No luck...

I'm a (very small) hosting company and I've been testing Postal in order to better understand and monitor my server's health, in terms of email deliverability.

Thanks

**Reply by willpower232** [reply]
Postals main intention is to send emails, yes you can route emails through it but as you see, it doesn't work for every use case.

If you have some code or a device which needs to send emails, then Postal helps a lot. If you're trying to do regular email things with Postal then you're going to struggle.
